microservices/product-search-export/config/services.yaml

```diff
@@ -9,16 +9,16 @@ services:
         public: false
 
     Elasticsearch\ClientBuilder:
-      class: Elasticsearch\ClientBuilder
-      factory: [Elasticsearch\ClientBuilder, create]
-      calls:
-        - method: setHosts
-          arguments:
-            - ['%env(ELASTICSEARCH_HOSTS_STRING)%']
+        class: Elasticsearch\ClientBuilder
+        factory: [Elasticsearch\ClientBuilder, create]
+        calls:
+            - method: setHosts
+            - arguments:
```

Comment:
Doesn't the added dash change the value? I think the calls not contains `[{method: setHosts}, {arguments: [['%env(ELASTICSEARCH_HOSTS_STRING)%']]}]` instead of `[{method: setHosts, arguments: [['%env(ELASTICSEARCH_HOSTS_STRING)%']]}]` (i.e. two objects in an array instead of one).

Reply:
you have right, i fixed a few file before a fix indent for array and this is badly fixed file
indent for file is now fixed right